core/src/main/resources/jenkins/model/Jenkins/configure.jelly

```diff
@@ -61,8 +64,8 @@ THE SOFTWARE.
       </j:forEach>
 
       <f:bottomButtonBar>
-        <f:submit value="${%Save}" />
-        <f:apply />
+        <f:submit value="${%Save All}"/>
+        <f:apply value="${%Apply All}"/>
```

Comment:
@tfennelly Yes. With everything on one page, no need to change label to clarify all tabs are getting saved.